common/xbps-src/shutils/chroot.sh

```diff
@@ -210,12 +210,14 @@ chroot_sync_repodata() {
     mkdir -p $confdir
     if [ -n "$XBPS_ALT_REPOSITORY" ]; then
         ( \
+            echo "repository=$hostdir/binpkgs/${XBPS_ALT_REPOSITORY}/bootstrap"; \
```

Comment:
While we're at it, can we change this to heredoc? subshell is not necessary, I think